mmseg/datasets/isprs.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from .builder import DATASETS
+from .custom import CustomDataset
+
+
+@DATASETS.register_module()
+class ISPRSDataset(CustomDataset):
+    """ISPRS dataset.
+
+    In segmentation map annotation for LoveDA, 0 is the ignore index.
```

Comment:
The docstring is for LoveDA.
Please correct it.

Reply:
Which dataset name is better, ISPRS or Potsdam?

Reply:
It is better to add Potsdam and Vaihingen dataset separately.
And one derives from another.
Refer to https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/dark_zurich.py#L6.

Reply:
Ok, i will try